packages/luna/src/components/TutorFeedback/RatingDialog.tsx

```diff
@@ -70,9 +77,12 @@ export const EditRating: React.FC<FeedbackEditProps> = ({
         <Button
           size={ButtonSize.Large}
           onClick={() =>
-            onUpdate({ value: newRating, feedback: newFeedback || null })
+            onSubmit({ value: newRating, feedback: newFeedback || "" })
```

Comment:
Incase no feedback, it should be null. 


```suggestion
            onSubmit({ value: newRating, feedback: newFeedback || null })
```